validator/src/bin/solana-test-validator.rs

```diff
@@ -325,6 +325,16 @@ fn main() {
                 .long("no-accounts-db-caching")
                 .help("Disables accounts caching"),
         )
+        .arg(
+            Arg::with_name("deactivate_feature")
+                .long("deactivate-feature")
+                .short("d")
```

Comment:
This looks great, thanks. My only nit is regarding camping on the `-d` short arg. I think I'd rather leave this out and only support the long `--deactivate-feature` form for now. `-d` is quite generic and maybe there'll be a more important arg in the future we'll want to use this for (and I fully recognize I set this precedent with `-c`!)

Reply:
`-df` seems decent. I never actually tried a `.short()` arg with more than one characters in it, lol it works though?

Reply:
It does not actually work :(